pkgs/development/python-modules/logica/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  duckdb,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "logica";
+  version = "1.3.14159265358";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-BC3pjy5jde/2MWvR2oMbli0C4GssV7hdIQ82Hpc2RAM=";
+  };
```

Comment:
add space and fetchFromGitHub is preferred

Reply:
The upstream project isn't buildable from github, only from pypi (e.g. there is no `setup.py` in the repo, but there is in the sdist from pypi). https://github.com/EvgSkv/logica

Space added

Reply:
put a comment please

Reply:
done